server/start.go

```diff
@@ -52,7 +53,6 @@ const (
 	FlagInterBlockCache    = "inter-block-cache"
 	FlagUnsafeSkipUpgrades = "unsafe-skip-upgrades"
 	FlagTrace              = "trace"
-	FlagTracing            = "tracing"
```

Comment:
nit, should we just keep it here to be consistent with all the other Flag constants?

Reply:
Putting it here requires an import of this file in baseapp.go which complains about an import cycle